src/JuMP.jl

```diff
@@ -329,6 +329,17 @@ function direct_model(backend::MOI.ModelLike)
     )
 end
 
+"""
+    direct_model(optimizer::MOI.OptimizerWithAttributes)
+
+Return a new JuMP model that instantiates a [`OptimizerWithAttributes`](@ref) to store the
```

Comment:
Mention [`optimizer_with_attributes`](@ref) instead of [`OptimizerWithAttributes`](@ref).

Something like this?
````Julia
"""
    direct_model(factory::MOI.OptimizerWithAttributes)

Create a [`direct_model`](@ref) using `factory`, a `MOI.OptimizerWithAttributes`
object created by [`optimizer_with_attributes`](@ref).

## Example

```julia
model = direct_model(
    optimizer_with_attributes(
        Gurobi.Optimizer, 
        "Presolve" => 0, 
        "OutputFlag" => 1,
    )
)
```
is equivalent to:
```julia
model = direct_model(Gurobi.Optimizer())
set_optimizer_attribute(model, "Presolve", 0)
set_optimizer_attribute(model, "OutputFlag", 1)
```
"""
````

Reply:
fixed via 1b8a6b1